docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,17 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+When you configure automated builds in Docker Cloud you can specify additional Environment Variables. The variables you add on the **Configure build settings** screen are used during the build process _only_.
+
+You can also declare environment variables in `hooks` files, however anything
```

Comment:
I would remove the first sentence. Instead I would say that the nice thing about build env vars is that they are accessible from your hooks without the need of storing them in your git repo.